README.md

```diff
@@ -13,5 +13,9 @@ yarn install
 ## Usage
 
 ```bash
+node localScrapper.js # Prepares karakuri-ready data from karaoke files
 npm start
-```
\ No newline at end of file
+```
+
+## Options
+Run `node localScrapper.js --help` and `node start.js --help` to get a list of options.
```

Comment:
Should be `npm run scrap` and `npm start -- --help`, `npm run scrap -- --help`
The main interest for npm run is that we can manipulate what it does much more flexibly.